app/components/home-page/organization-dashboard/side-nav/index.hbs

```diff
@@ -0,0 +1,92 @@
+<aside local-class='side-menu-root'>
+  <div local-class='app-logo'>
+    <ImgLogo />
+  </div>
+
+  <AkList class='mt-2' as |akl|>
+    {{#each this.menuItems as |it|}}
+      <akl.listItem
+        @link={{true}}
+        @route={{it.route}}
+        @query={{it.query}}
+        @currentWhen={{it.currentWhen}}
+        @activeLinkClass={{this.classes.menuItemLinkActive}}
+        data-test-side-menu-item
+        @linkClass={{this.classes.menuItemLink}}
+        as |li|
+      >
+        <li.leftIcon>
+          <AkIcon
+            local-class='menu-item-icon'
+            data-test-side-menu-item-icon
+            @iconName={{it.icon}}
+          />
+        </li.leftIcon>
+
+        <li.text
+          data-test-side-menu-item-text
+          @primaryText={{it.label}}
+          @primaryTextClass={{this.classes.menuItemText}}
+        />
+
+        {{#if it.hasBadge}}
+          <AkStack @alignItems='center' @justifyContent='center'>
+            <AkChip
+              data-test-side-menu-item-badge
+              local-class='menu-item-badge'
+              @variant='filled'
+              @size='small'
+              @label='
```

Comment:
I think it would be much better if the label is part of the `it` object. We could have more labels for other list items that have badges.

Reply:
done